dual-gauge-card.js

```diff
@@ -126,13 +126,17 @@ class DualGaugeCard extends HTMLElement {
 
   _getEntityStateValue(entity, attribute) {
     if (!attribute) {
-      return entity.state;
+      if(isNaN(entity.state)) return "-" ; //check if entity state is NaN
+      else return (Math.round(entity.state*10)/10); //rounds the value to 1 decimal place
```

Comment:
```suggestion
      else return entity.state;
```
I don't think the value should be rounded by default. Maybe you make the number of decimals configurable?

Reply:
Well then, there should be an option to set the precision from the frontend? I assumed that 1 decimal place allows the values to be shown properly in the card and not be squashed.

Reply:
Yes, there should be. And maybe there will be sometime. But for now, lets finally merge the NaN part.